gapic/src/main/com/google/gapid/models/Experiments.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.models;
+
+public class Experiments {
+  public static final boolean ShouldEnableExperiments = false;
+
+  boolean shouldDisableAF;
```

Comment:
remove the `should` and don't abbreviate anisotropic filtering. The getter would still be `shouldDisable...`, but the field does not need it.